tests/unittests/JITTest.cpp

```diff
@@ -561,6 +561,20 @@ TEST_P(BackendCorrectnessTest, smallConv) {
   EXPECT_TRUE(out1.isEqual(out2));
 }
 
+/// This test targets the DKKC8 optimization.
+TEST_P(CPUOnly, groupConvTest) {
+  std::array<size_t, 4> S{{1, 2, 1, 128}};
+  llvm::ArrayRef<size_t> shape(S);
+  Tensor out1(ElemKind::FloatTy, shape);
+  Tensor out2(ElemKind::FloatTy, shape);
+  inferGroupConv(&out1, backendKind_);
+  inferGroupConv(&out2, BackendKind::Interpreter);
+
+  for (size_t i = 0; i < 256; i++) {
```

Comment:
any reason not to EXPECT_TRUE(out1.isEqual(out2)); ?

Reply:
I used isEqual. For release mode, it works. For debug_mode, it reported false. Tried to output all the elements for debug_mode, but it looks to me the values are equal. Just put  this version here first and still checking isEqual.

Reply:
This means that there's very small difference between numbers, e.g. digit number 6 after floating point. Release and debug mode are expected to be a bit different sometimes, because of fast-math optimizations etc.

Perhaps need to adjust allowed error for `isEqual()`.

Reply:
Yup, as Artem said you can tune, float allowedError = 0.0001.